ReactNative/ReactTwitter/core/views/splash-screen.js

```diff
@@ -49,7 +50,7 @@ var SplashScreenView = React.createClass({
   render () {
     return (
       <View style={styles.container}>
-        <Text style={styles.title}>SplashScreen</Text>
+        <Image style={styles.logo} source={require('./assets/twitter-logo.png')} />
```

Comment:
We can use images with differents densities using the suffix: @2x and @3x. 

From the documentation: 

> Packager will bundle and serve the image corresponding to device's screen density, e.g. on iPhone 5s check@2x.png will be used, on Nexus 5 – check@3x.png. If there is no image matching the screen density, the closest best option will be selected.